be-java-quarkus/Jenkinsfile

```diff
@@ -0,0 +1,80 @@
+def odsNamespace = env.ODS_NAMESPACE ?: 'ods'
+def odsGitRef = env.ODS_GIT_REF ?: 'master'
+def odsImageTag = env.ODS_IMAGE_TAG ?: 'latest'
+
+library("ods-jenkins-shared-library@${odsGitRef}")
+
+odsQuickstarterPipeline(
+    imageStreamTag: "${odsNamespace}/jenkins-slave-maven:${odsImageTag}",
```

Comment:
Couldn't this use the GraalVM Jenkins agent?

Reply:
good idea, that way I could just install the latest maven version there 😮 
Will try it.

Reply:
worked nicely